modules/airgridRtdProvider.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * This module adds the AirGrid provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ * The module will fetch real-time audience data from AirGrid
+ * @module modules/airgridRtdProvider
+ * @requires module:modules/realTimeData
+ */
+import {config} from '../src/config.js';
+import {submodule} from '../src/hook.js';
+import {mergeDeep, isPlainObject, deepSetValue, deepAccess} from '../src/utils.js';
+import {getGlobal} from '../src/prebidGlobal.js';
+import {getStorageManager} from '../src/storageManager.js';
+
+const MODULE_NAME = 'realTimeData';
+const SUBMODULE_NAME = 'airgrid';
+const AG_TCF_ID = 782;
+export const AG_AUDIENCE_IDS_KEY = 'edkt_matched_audience_ids'
+
+export const storage = getStorageManager(AG_TCF_ID, SUBMODULE_NAME);
+
+/**
+ * Attach script tag to DOM
+ * @param {Object} rtdConfig
+ * @return {void}
+ */
+export function attachScriptTagToDOM(rtdConfig) {
+  var edktInitializor = window.edktInitializor = window.edktInitializor || {};
+  if (!edktInitializor.invoked) {
+    edktInitializor.invoked = true;
+    edktInitializor.accountId = rtdConfig.params.accountId;
+    edktInitializor.publisherId = rtdConfig.params.publisherId;
+    edktInitializor.apiKey = rtdConfig.params.apiKey;
+    edktInitializor.load = function(e) {
+      var p = e || 'sdk';
+      var n = document.createElement('script');
+      n.type = 'text/javascript';
+      n.async = true;
+      n.src = 'https://cdn.edkt.io/' + p + '/edgekit.min.js';
+      var a = document.getElementsByTagName('head')[0];
+      a.parentNode.insertBefore(n, a);
```

Comment:
This appears to place the `<script>` tag outside of and before the `<head>` tag. Shouldn't the script be loaded within `<head>`?

Reply:
@msm0504 yeah good catch thank you!

I have used `nextSibling` on the reference node to ensure we insert after the `head` tag.

Reply:
Shouldn't the `script` tag be inserted inside of the `head` tag?